source/test-suite/Main.hs

```diff
@@ -635,7 +635,12 @@ main = Hspec.hspec . Hspec.parallel . Hspec.describe "cabal-gild" $ do
     Hspec.it "formats impl without version range" $ do
       expectGilded
         "if impl ( ghc )"
-        "if impl(ghc >=0)\n"
+        "if impl(ghc)\n"
```

Comment:
This changed because Cabal used to insert an implicit `anyVersion` whenever the version range was missing. Now the optionality is properly modeled.